cairo/tests/utils/args_gen.py

```diff
@@ -1156,6 +1156,8 @@ def default_factory():
             return default_value
 
         data = defaultdict(default_factory, data)
+    else:
+        default_value = 0
```

Comment:
this is not correct - we can't assume that the default value is actually zero. for a trie[bytes32, u256] the default value needs to pointer to u256(0,0), and because the trie strategies are not generating default dicts, this will cause problems

Reply:
if anything we probably need to assert that the dict passed is a default dict

Reply:
the fix is wrong and was done to rapidly 
lookin into it now